terraform/modules/monitoring/policy.tf

```diff
@@ -0,0 +1,30 @@
+resource "aws_sns_topic_policy" "this" {
```

Comment:
I could have also just used regular IAM policies here. See [here](https://docs.aws.amazon.com/sns/latest/dg/sns-using-identity-based-policies.html#iam-and-sns-policies) for a comparison of the approaches. In retrospect I think it might actually have been better to use regular IAM policies (rather than `aws_sns_topic_policy`) - like how it was done in EPB. The way I've done it here does allow for cross-account SNS publishing, but I don't think that will be needed.

I wanted to get a second opinion though about whether the other way was indeed better, and even if it was whether it was worth the time to change it (probably yes IMO).

Reply:
Overall I don't think it matters - the main concern is probably how easy it is to add new things that can publish to the topic (particularly for e.g. review apps where we might end up with one topic and several instances of other things publishing notifications)

Reply:
Makes sense to me